src/Microsoft.AspNetCore.Razor.Language/RazorProjectEngine.cs

```diff
@@ -20,60 +20,45 @@ public abstract class RazorProjectEngine
 
         public static RazorProjectEngine Create(RazorProjectFileSystem fileSystem) => Create(fileSystem, configure: null);
 
-        public static RazorProjectEngine Create(RazorProjectFileSystem fileSystem, Action<RazorProjectEngineBuilder> configure)
+        public static RazorProjectEngine Create(RazorProjectFileSystem fileSystem, Action<RazorProjectEngineBuilder> configure) => Create(fileSystem, configure, RazorConfiguration.Default);
+
+        public static RazorProjectEngine Create(
+            RazorProjectFileSystem fileSystem,
+            Action<RazorProjectEngineBuilder> configure,
+            RazorConfiguration configuration)
```

Comment:
Can we put the `Action<>` last? That will look much cleaner to use

Reply:
sure.